src/routes/github/repository/github-repository-get.ts

```diff
@@ -27,7 +33,7 @@ export const GitHubRepositoryGet = async (req: Request, res: Response): Promise<
 			repositories
 		});
 	} catch (err) {
-		req.log.error({ err }, "Error searching repository");
+		req.log.error({ err }, "Error fetching repositories");
```

Comment:
A little hard to tell in our logs which of these 2 errors we're hitting when they say exactly the same thing. Updated the wording for clarity.